### DISCUSSION TITLE
Issues with file synchronization on Nextcloud AIO v29.0.6

### DISCUSSION THREAD

**Discussion by vawaver** [nextcloud/all-in-one#5318] [Questions] [answered]
## Issues with file synchronization on Nextcloud AIO v29.0.6 (Ubuntu Server 24.04)

Greetings,

I am using Nextcloud AIO version 29.0.6, hosted on a server running Ubuntu 24.04. I am experiencing issues while trying to synchronize files from the server to my new local desktop.

The synchronization works fine on another device, but on this new desktop, I encounter errors when trying to sync multiple files. I have attached a screenshot showing the specific errors I am receiving.

Here are the details:
- **Nextcloud AIO version**: 29.0.6
- **Server OS**: Ubuntu 24.04
- **Issue**: Errors during file synchronization (see attached screenshot)
- **Other devices**: Files are syncing correctly on a different device

Could you please advise on how to resolve this issue? Any guidance or troubleshooting steps would be greatly appreciated.

![nextcloud-issue_small](https://github.com/user-attachments/assets/c1194ec1-e30f-42cb-9c74-1072a2b20202)

**Comment by szaimen** [comment, marked as answer]
Hi, this is a bug in the desktop client v3.14.0: https://github.com/nextcloud/desktop/issues/7136
